mlflow/models/container/scoring_server/nginx.conf

```diff
@@ -22,13 +22,14 @@ http {
     listen 8080 deferred;
     client_max_body_size 5m;
 
-    keepalive_timeout 5;
+    keepalive_timeout 65;
```

Comment:
Why not just change this to the default value of 75?

Reply:
I agree. The default value of 75 will also work. Let me make the code change. Thanks!